packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.base.tsx

```diff
@@ -1,39 +1,43 @@
 import * as React from 'react';
 import {
   BaseComponent,
-  css,
   getRTL,
-  createRef
+  createRef,
+  classNamesFunction,
+  IClassNames
 } from '../../Utilities';
 import { FocusZone, FocusZoneDirection } from '../../FocusZone';
 import { Link } from '../../Link';
 import { Icon } from '../../Icon';
 import { IconButton } from '../../Button';
+import { customizable } from '../../Utilities';
 import { IBreadcrumbProps, IBreadcrumbItem } from './Breadcrumb.types';
 import { DirectionalHint } from '../../common/DirectionalHint';
 import { ResizeGroup } from '../../ResizeGroup';
 import { TooltipHost, TooltipOverflowMode } from '../../Tooltip';
+import { IBreadcrumbStyleProps, IBreadcrumbStyles } from './Breadcrumb.styles';
 
-import * as stylesImport from './Breadcrumb.scss';
-const styles: any = stylesImport;
+const getClassNames = classNamesFunction<IBreadcrumbStyleProps, IBreadcrumbStyles>();
 
 export interface IBreadCrumbData {
   props: IBreadcrumbProps;
   renderedItems: IBreadcrumbItem[];
   renderedOverflowItems: IBreadcrumbItem[];
 }
 
-const OVERFLOW_KEY = 'overflow';
-const nullFunction = (): null => null;
+const OverflowKey = 'overflow';
```

Comment:
❓

Reply:
I admit the casing of const values is a bit unclear.

Example:
`const FontSize = { small: 16, etc };`
`const FONT_SIZE = { small: 16, etc };`

Usage of `FontSize.small` feels correct, moreso than FONT_SIZE.small.

What if we adjust the standards here;

TitleCasing for file scoped consts that represent data values.
camelCasing for file scoped functions.

Reply:
I previously did a survey before of JS OSS code and found that all caps separated by underscore (FONT_SIZE) is consistently used in JS source code for symbolic constants. The key thing here is "symbolic constants" and agree with you that `FONT_SIZE.small` is weird - but that's because in this case FONT_SIZE is not a symbolic constant if it is `{ small: 16, etc. }`.

I would advocate for sticking to `FONT_SIZE` for symbolic constants, but otherwise pascal casing for file scoped consts that represents data objects and camel casing for file scoped functions :)